samples/go/referenceapp/README.md

```diff
@@ -91,5 +91,19 @@ To use the DynamoDB Metastore included with Asherah, the following table should
    ReadCapacityUnits=1,WriteCapacityUnits=1
 ```
```

Comment:
We can mention in the above example that in case of DynamoDb, you can use a custom table name on L:83